resources/setups/baseline/module_config.json

```diff
@@ -1,10 +1,13 @@
 {
   "modules": {
     "ipaddr": {
-      "enabled": false,
-      "timeout_sec": 900,
+      "enabled": true,
+      "timeout_sec": 2000,
       "port_flap_timeout_sec": 20,
-      "dhcp_ranges": [{"start": "192.168.0.1", "end": "192.168.255.254", "prefix_length": 16}]
+      "dhcp_ranges": [{"start": "192.168.0.1", "end": "192.168.255.254", "prefix_length": 16},{"start": "10.0.0.1", "end": "10.255.255.254", "prefix_length": 8},{"start": "172.16.0.1", "end": "172.31.255.254", "prefix_length": 12}]
```

Comment:
can you split the 3 different ranges onto 3 different lines?
"dhcp_ranges": [
  {"start"... },
  {"start"... },
  {"start"... },
]